spec/unit/compliance_engine/compliance_engine_enforce_spec.rb

```diff
@@ -28,91 +28,103 @@
     ] + expected_classes.map{|c| Regexp.new("^(?!#{c}(::.*)?)")}
   }
 
+  valid_systems = {
+    'CentOS'      => '7',
+    'RedHat'      => '7',
+    'OracleLinux' => '7'
+  }
```

Comment:
Would be nice if simp-rake-helpers had a mapping of osfamily to OS.  Then you could read the compliance checks.yaml  and map it to a set of valid OSs instead of hard coding them.